packages/smooth_app/lib/pages/folksonomy/folksonomy_create_edit_modal.dart

```diff
@@ -32,12 +37,18 @@ class FolksonomyEditTagContentState extends State<FolksonomyEditTagContent> {
   late TextEditingController valueController;
   bool isKeyValid = true;
   bool isValueValid = true;
+  late final FocusNode keyFocusNode;
+  late final FocusNode valueFocusNode;
+  final Key keyAutocompleteKey = UniqueKey();
+  final Key valueAutocompleteKey = UniqueKey();
```

Comment:
`FocusNode` and `Key` are initialized the same way.
So please decide, if it's in `initState` or directly here